### PR TITLE
Can update entry

### PR DESCRIPTION
**Pull request by johndo-dev**

### CODE REVIEW

**Comment by jdo697** [comment]
image
image

**Comment by jdo697** [comment]
image

File: main.js [hunk 1]
```diff
@@ -1,46 +1,219 @@
...
+  Thursday: [],
+  Friday: [],
+  Saturday: []
+};
+
+var todoList = {
+  addEntry: function(day, newTask) {
+    if (schedule[day] === undefined) {
+      console.log('schedule[day] is undefined');
+    }
```
Comment (uzair-ashraf) [inline main.js:15]: return if its undefined I'm guessing

File: main.js [hunk 2]
```diff
@@ -1,46 +1,219 @@
...
+    entrySubmit.addEventListener('click', function (event) {
+      event.preventDefault();
+
+      var newTask = {
+        day: entryWeek.value,
+        time: entryTime.value,
+        description: entryDescription.value,
+      };
+
+      todoList.addEntry(entryWeek.value, newTask);
```
Comment (uzair-ashraf) [inline main.js:46]: If the input is empty, then you're going to get undefined, just make sure its not undefined

**Review by uzair-ashraf — CHANGES_REQUESTED** [review]

**Comment by jdo697** [comment]
F1
F2
F3

**Comment by jdo697** [comment]
Untitled

**Pull request merged**